frontend/components/commons/header/global-actions/text-classification-bulk-annotation/textClassificationBulkAnnotation.spec.js

```diff
@@ -0,0 +1,151 @@
+import { shallowMount } from "@vue/test-utils";
```

Comment:
the whole text-classifier-bulk-annotation folder is better to be outside of commons, in the textClassifier folder